src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[][] data = readData(fromFileName);
+
+        String resultData = calculateStatistics(data);
+
+        writeData(toFileName, resultData);
```

Comment:
The method `getStatistic` is well-decomposed and follows the single responsibility principle. However, the method name could be more descriptive to reflect that it reads, processes, and writes data, such as `processAndWriteStatistics`.